assets/js/main.js

```diff
@@ -156,6 +156,32 @@ $(document).ready((function ($) {
         return false;
     });
 
+    $('.appointment-button').click(function () {
+        var $form = $('.appointment-form'),
+            data = getFormData($form),
+            host = 'http://127.0.0.1:8000/',
+            path = 'api/appointment/',
+            url = host + path;
+
+        $.ajax({
+            type: 'POST',
+            url: url,
+            data: JSON.stringify(data),
+            beforeSend: removeFieldErrors,
+            success: function () {
+                $('#section .alert-success').fadeIn(function () {
```

Comment:
`#section .alert-success` should change to `.alert-success` after fix for comment above